pages/linux/fprintd-verify.md

```diff
@@ -0,0 +1,28 @@
+# fprintd-verify
+
+> Verify fingerprints against the database.
+> More information: <https://manned.org/fprintd-verify>.
+
+- Verify all stored fingerprints for the current user:
+
+`fprintd-verify`
```

Comment:
Does this require the user to verify all their fingers in turn, or is just any single finger enough?

Reply:
The user will be asked to verify finger by finger.